packages/dev/core/src/Cameras/Inputs/freeCameraDeviceOrientationInput.ts

```diff
@@ -52,6 +56,8 @@ export class FreeCameraDeviceOrientationInput implements ICameraInput<FreeCamera
     private _beta: number = 0;
     private _gamma: number = 0;
 
+    public smoothFactor: number = 0;

```

Comment:
Need comment for public variables

Reply:
comment added for addDeviceOrientation() param, but I feel public variable comment has already been added